habitat-lab/habitat/datasets/rearrange/samplers/receptacle.py

```diff
@@ -506,6 +506,25 @@ def import_tri_mesh(mesh_file: str) -> List[mn.trade.MeshData]:
     importer = manager.load_and_instantiate("AnySceneImporter")
     importer.open_file(mesh_file)
 
+    def filter_interleave_mesh(mesh: mn.trade.MeshData) -> mn.trade.MeshData:
+        """
+        Filter all but position data and interleave a mesh to reduce overall memory footprint.
+        """
+        # `mesh_filtered` now refers to data in `mesh`
+        # (`mesh_filtered.vertex_data_flags` isn't `OWNED`)
+        # until a proper fix, it's important to use a different variable
```

Comment:
I have the fix almost done, just need to wait until my CIs pass :)

If you can hold off with merging this until tomorrow, I should manage to open a PR upgrading the relevant repos on the sim side by then. Possibly even including the `Flag::Quiet` on the importers (which seems easier from the two options I outlined).

Reply:
Yeah, I think that sounds reasonable. Aim to merge this tomorrow with remaining improvements. Thanks for the support here. :slightly_smiling_face: